apps/dashboard/src/app/account/billing/usage/usage-table.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { AccountUsage, InvoiceItem } from '../billing.models';
+
+@Component({
+  selector: 'app-usage-table',
+  templateUrl: './usage-table.component.html',
+  styleUrls: ['./usage-table.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class UsageTableComponent {
+  paramsToShow = ['media', 'paragraphs', 'searches', 'qa', 'training-hours'];
+
+  @Input()
+  set usage(value: AccountUsage | undefined) {
+    if (value) {
+      this._usage = value;
+      this.invoiceItems = Object.entries(value.invoice_items)
+        .filter(([type]) => this.paramsToShow.includes(type))
+        .sort(([type1], [type2]) => this.paramsToShow.indexOf(type1) - this.paramsToShow.indexOf(type2));
+
+      this.total = this.invoiceItems.reduce((acc, [, item]) => acc + item.over_cost, 0);
+    }
+  }
+  get usage() {
+    return this._usage;
+  }
+  _usage?: AccountUsage;
```

Comment:
When using accessors, the accessed property should be private: `private _usage?: AccountUsage`. Actually the `_` prefix is usually used to mark private properties

Reply:
Fixed